cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -320,6 +322,7 @@ public static void sanityCheckCapacityEstimation(boolean allowCapacityEstimation
       throws KafkaCruiseControlException {
     PreferredLeaderElectionGoal goal = new PreferredLeaderElectionGoal();
     try (AutoCloseable ignored = _loadMonitor.acquireForModelGeneration(operationProgress)) {
+      sanityCheckBrokerPresence(brokerIds);
```

Comment:
Can we move `sanityCheckBrokerPresence(brokerIds);` out of the try-with-resources statement?